README.md

```diff
@@ -49,6 +49,48 @@ r := wazero.NewRuntimeWithConfig(wazero.NewRuntimeConfigInterpreter())
 
 The below support policy focuses on compatability concerns of those embedding wazero into their Go applications.
 
+### WebAssembly
+
+#### Core API
+wazero supports the only WebAssembly specification which has reached W3C
+Recommendation (REC) phase: [WebAssembly Core Specification 1.0 (20191205)](https://www.w3.org/TR/2019/REC-wasm-core-1-20191205).
+
+Independent verification is possible via the [WebAssembly spec test suite](https://github.com/WebAssembly/spec/tree/wg-1.0/test/core),
+which we run on every change and against all supported platforms.
+
+One current limitation of wazero is that it doesn't fully implement the Text
+Format, yet, e.g. compiling `%.wat` files. The intent is to finish this, and
+meanwhile users can work around this using tools such as `wat2wasm` to compile
+the text format into the binary format.
+
+#### Post 1.0 Features
+
+The last W3C REC was at the end of 2019. There were other features that didn't
+make the cut or were developed in the years since. The community unofficially
+refers to these as [Finished Proposals](https://github.com/WebAssembly/proposals/blob/main/finished-proposals.md).
+
+To ensure compatability, wazero does not opt-into any non-standard feature by
```

Comment:
`opt-into` is not in a dictionary, so it may be a little bit difficult to guess the meaning for non-native English readers (yeah, like me). Maybe the same meaning as "opt-in"?

Reply:
missing space :D

Reply:
thanks for looking close enough to notice, and thinking to say something. fixed!